_studio/shared/umc/codec/av1_dec/src/umc_av1_decoder.cpp

```diff
@@ -544,7 +544,7 @@ namespace UMC_AV1_DECODER
 
         UMC::Status umcRes = SubmitTiles(*pCurrFrame, firstSubmission);
         if (umcRes != UMC::UMC_OK)
-            return umcRes;
+            return UMC::UMC_ERR_FAILED;
```

Comment:
This might break something. Can you go inside SubmitTiles and find where you get the error?

Reply:
This call returns error from driver:
https://github.com/Intel-Media-SDK/MediaSDK/blob/49f1e2907683c76f1043b2672e223be09af5f178/_studio/shared/umc/codec/av1_dec/src/umc_av1_decoder_va.cpp#L91-L93
Added mapping to UMC_ERR_FAILED.